rust/src/builder.rs

```diff
@@ -153,6 +169,12 @@ impl DeltaTableBuilder {
         self
     }
 
+    /// Sets `log_buffer_size` to the builder
+    pub fn with_log_buffer_size(mut self, log_buffer_size: usize) -> Self {
```

Comment:
```suggestion
    pub fn with_buffer(mut self, log_buffer_size: usize) -> Self {
```

IMHO this can be more concise